app/serializers/rest/keybase_proof_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class REST::KeybaseProofSerializer < ActiveModel::Serializer
+  attributes :token
```

Comment:
this should be `sig_hash` since the client/servers expect to consume that value

Reply:
correct